setup.cfg

```diff
@@ -44,6 +44,7 @@ docs =
     furo
     sphinx
     sphinx_design
+test = tox
```

Comment:
Thanks for making this change. Rather than adding a new extra, please could you install tox directly in the `postCreateCommand`. Ultimately I want to get rid of the `docs` extra as well because I don't think that it's a good practise to put development requirements in extras.